Libraries/Components/TextInput/RCTTextInputViewConfig.js

```diff
@@ -148,6 +148,7 @@ const RCTTextInputViewConfig = {
     clearTextOnFocus: true,
     showSoftInputOnFocus: true,
     autoFocus: true,
+    ios_lineBreakStrategy: true,
```

Comment:
I think our convention is `lineBreakStrategyIOS` -- can we update that to this

from @javache 
> I believe we generally suffix the platform for platform-specific props (eg shouldRasterizeIOS), or don't suffix them at all, and document that the prop is android-only (eg focusable)

Reply:
I'll change it. thanks!